web/modules/custom/collection/collection.behat.inc

```diff
@@ -111,7 +111,7 @@ class CollectionSubContext extends DrupalSubContextBase implements DrupalSubCont
   /**
    * Navigates to the collections overview page.
    *
-   * @When (I )visit the collection overview page
+   * @When (I )visit the collection overview( page)
```

Comment:
The (I) should be outside the parenthesis as it does not make sense otherwise.

Reply:
Yes yes yes :D I made a PR for this https://github.com/ec-europa/joinup-dev/pull/1554